### PR TITLE
Removing a period

### PR DESCRIPTION
**Pull request by bmallred**
Resolves truetandem/e-QIP-prototype#2479

### CODE REVIEW

File: src/config/locales/en/application.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const application = {
       'Not a guarantee of acceptance, but all required fields are complete.'
     ],
     valid2: [
-      '### Please sign the releases below and submit your form.'
```
Comment (floresj) [inline src/config/locales/en/application.js]: There's no shelter here `period`

Comment (bmallred) [reply src/config/locales/en/application.js]: We should ramble on with this PR to increase its visibility by the ever important "conversation" metric.

Comment (floresj) [reply src/config/locales/en/application.js]: .cirtem "noitasrevnoc" tnatropmi reve eht yb ytilibisiv sti esaercni ot RP siht htiw no elbmar dluohs eW.cirtem "noitasrevnoc" tnatropmi reve eht yb ytilibisiv sti esaercni ot RP siht htiw no elbmar dluohs eW.

Comment (floresj) [reply src/config/locales/en/application.js]: All my rambling and I forget to actually approve this thing.

Comment (bmallred) [reply src/config/locales/en/application.js]: I approve your approval to approve

Comment (floresj) [reply src/config/locales/en/application.js]: Your approval to approve my approval request shows your willingness to approve items that require approval is incredibly high.

Comment (bmallred) [reply src/config/locales/en/application.js]: image

**Pull request merged**